processor/src/main/java/com/ihren/processor/dto/ItemDto.java

```diff
@@ -0,0 +1,16 @@
+package com.ihren.processor.dto;
+
+import com.ihren.processor.validation.contains.in.ContainsIn;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+public record ItemDto(
+        @NotNull
+        Long id,
+        @ContainsIn({"1", "2", "3", "4"})
```

Comment:
It will not be convinient to put values from dto. Sometimes it will be necessary to be received from application.yaml.
So, You should specify annotation for special case